tests/runner.js

```diff
@@ -3,5 +3,24 @@
 
 import 'babel-polyfill';
 
+if (process.env.COVERAGE === 'enabled') {
+  // generate the coverage reports
+  after((done) => {
+    let istanbul = require('babel-istanbul');
+    let collector = new istanbul.Collector();
+    let reporter = new istanbul.Reporter();
+    let sync = false;
+
+    collector.add(global.__coverage__);
+
+    reporter.add('text');
+    reporter.addAll([ 'lcov', 'clover' ]);
+    reporter.write(collector, sync, function() {
+      console.log('All reports generated');
+      done();
```

Comment:
That is interesting, I didn't know about the coveralls service. 
I'm looking into it, it seems that it would be possible to integrate it by just sent to coveralls the coverage results in the lcov format, e.g.:

```
npm install coveralls && cat ./coverage/lcov.info | coveralls
```